types/connect/index.d.ts

```diff
@@ -18,11 +18,15 @@ declare function createServer(): createServer.Server;
 declare namespace createServer {
     export type ServerHandle = HandleFunction | http.Server;
 
+    export class IncomingMessage extends http.IncomingMessage {
```

Comment:
This should be "implements" as http.IncomingMessage is an interface.

My code will no longer compile in typescript 3.7.2 because of this issue.

Full error is "node_modules/@types/connect/index.d.ts(21,42): error TS2689: Cannot extend an in
terface 'http.IncomingMessage'. Did you mean 'implements'?"

Please can we fix this ASAP.

Reply:
According to [the types in this repo](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/http.d.ts#L269), `http.IncomingMessage` is a class. Where are you getting your types from?

Reply:
Ah thanks for that. Turns out we were using an older version of node @types where the `http.IncomingMessage` was an interface. Thanks for your prompt response